projects/common/src/time/time-range.service.ts

```diff
@@ -64,14 +64,23 @@ export class TimeRangeService {
   }
 
   private initializeTimeRange(): void {
-    this.navigationService.navigation$
+    combineLatest([
+      this.navigationService.navigation$,
+      this.navigationService.navigation$.pipe(switchMap(activatedRoute => activatedRoute.queryParamMap))
+    ])
       .pipe(
-        take(1), // Wait for first navigation
-        switchMap(activatedRoute => activatedRoute.queryParamMap), // Get the params from it
-        take(1), // Only the first set of params
-        map(paramMap => paramMap.get(TimeRangeService.TIME_RANGE_QUERY_PARAM)), // Extract the time range value from it
-        filter((timeRangeString): timeRangeString is string => !isEmpty(timeRangeString)), // Only valid time ranges
-        map(timeRangeString => this.timeRangeFromUrlString(timeRangeString)),
+        map(([activeRoute, paramMap]) => {
+          const defaultPageTimeRange = activeRoute.snapshot.data?.defaultTimeRange;
```

Comment:
Why does it need to read defaultTimeRange here? Can it not just read from url and init accordingly?